ideascube/static/ideascube/vendor/minislate/minislate.min.css

```diff
@@ -1,137 +0,0 @@
-.editor-toolbar {
```

Comment:
So if you're removing minislate, you probably should remove it from the first line of `ideascube.js` (I think that's a comment to help drive `eslint`... Oh wow! I had completely forgotten about that! Here's something to add to the CI!)